test/jdk/java/awt/Checkbox/CheckboxIndicatorSizeTest.java

```diff
@@ -119,21 +119,21 @@ private Frame createAndShowUI() {
         Checkbox cbEnglishOnly
             = new Checkbox("Toggle", true);
         Checkbox cbDingbatsOnly
-            = new Checkbox("\u274a\u274b\u274c\u274d", true);
+            = new Checkbox("❊❋❌❍", true);
         Checkbox cbEnglishDingbats
-            = new Checkbox("Toggle \u274a\u274d", true);
+            = new Checkbox("Toggle ❊❍", true);
         Checkbox cbDingbatsEnglish
-            = new Checkbox("\u274a\u274d toggle", true);
+            = new Checkbox("❊❍ toggle", true);
 
         CheckboxGroup radioGroup = new CheckboxGroup();
         Checkbox rbEnglishOnly
             = new Checkbox("Radio", true, radioGroup);
         Checkbox rbDingbatsOnly
-            = new Checkbox("\u274a\u274b\u274c\u274d", false, radioGroup);
+            = new Checkbox("❊❋❌❍", false, radioGroup);
```

Comment:
I'm not sure if this really looks better than the Unicode sequences?

Reply:
Neither am I. I was the verge on skipping this. This was one of the examples I was thinking about when saying "opinions may vary". I have no problems reverting this.